src/Magento/FunctionalTestingFramework/StaticCheck/TestDependencyCheck.php

```diff
@@ -62,6 +62,12 @@ class TestDependencyCheck implements StaticCheckInterface
      */
     private $alreadyExtractedDependencies;
 
+    /**
+     * Array containing all errors found after running the execute() function.
+     * @var array
+     */
+    private $errors;
```

Comment:
This new variable reveals that the original deisgn of the `TestDependencyCheck` should have been a little different.

* `execute()` should not return anything
* Class should have another new variable `private $output` so it can be accessible like `$staticCheck->getOutput()`

This is more future static check friendly, as it opens up the window for more nuanced error tracking/etc.

Keep in mind, you'll need to also edit the `StaticCheckInterface` but it's a simple edit.